test/app/dftb+/solvers/Si384_ELPA2/dftb_in.hsd

```diff
@@ -399,7 +399,7 @@ Hamiltonian = DFTB {
     Si = "p"
   }
   EigenSolver = ELPA {
-    Mode = 1
```

Comment:
Any reason for the change (as it's out of scope for the PR description)?

Reply:
Sorry that this appears in this merge request. I fixed this, because the folder name `Si384_ELPA2` suggests that it should be ELPA 2. Otherwise, the test would be identical to `Si384_ELPA1`.

Reply:
Can you just separate it off as a different PR, and squash out the change from this one?

Reply:
Yes, sure, done.